testing/test_base.sh

```diff
@@ -33,90 +35,12 @@ cat inst/result.log | tee -a $TEST_RESULTS
 
 echo %%%%%%%%%%%%%%%%%%%%%% Telnet fail | tee -a $TEST_RESULTS
 # Check that an open port causes the appropriate failure.
-docker rmi daqf/test_hold:latest # Check case of missing image
+docker rmi -f daqf/test_hold:latest # Check case of missing image
```

Comment:
why is -f necessary?  The CI process should be fairly deterministic...

Reply:
Well, looks like it goes through without -f. Will let it run a few times and see if the problem shows up again.